src/ReconcileWindow.cpp

```diff
@@ -194,6 +194,12 @@ ReconcileWindow::ReconcileWindow(const BRect frame, Account *account)
 			.Add(fReconcile, 7, 1)
 		.End()
 	.End();
+	
+	float maxWidth;
+	float maxHeight;
+	GetSizeLimits(NULL, &maxWidth, NULL, &maxHeight);
+
+	SetSizeLimits(frame.Width(), maxWidth, frame.Height(), maxHeight);
```

Comment:
Instead, add `B_AUTO_UPDATE_SIZE_LIMITS` to [Line 45](https://github.com/HaikuArchives/CapitalBe/blob/107651f3b5e5385db01d8876a7583f4660848c7e/src/ReconcileWindow.cpp#L45).